configuration/config-txt/overclocking.md

```diff
@@ -114,7 +114,7 @@ To view the Pi's current frequency, type: `cat /sys/devices/system/cpu/cpu0/cpuf
 
 To view the Pi's temperature, type `cat /sys/class/thermal/thermal_zone0/temp`. Divide the result by 1000 to find the value in degrees Celsius. Alternatively, there is a vcgencmd, `vcgencmd measure_temp` that interrogates the GPU directly for its temperature.
 
-Whilst hitting the temperature limit is not harmful to the SoC, it will cause CPU throttling. A heatsink can help to control the core temperature and therefore performance. This is especially useful if the Pi is running inside a case. Airflow over the heatsink will make cooling more efficient. A suitable heatsink is the self-adhesive BGA (ball-grid-array) 14x14x10 mm heatsink available from [RS Components](http://uk.rs-online.com/web/p/heatsinks/6744756/).
+Whilst hitting the temperature limit is not harmful to the SoC, it will cause CPU throttling. A heatsink can help to control the core temperature and therefore performance. This is especially useful if the Pi is running inside a case. Airflow over the heatsink will make cooling more efficient.
```

Comment:
Turns out this particular link was correct and can even be upgraded to HTTPS. It was just some temporary server blip when running the checker. So, <https://uk.rs-online.com/web/p/heatsinks/6744756/> can be used if needed.

Reply:
It's probably irrelevant anyway - heatsinks aren't really needed for the original Pi, and there are now _many_ 3rd-party heatsinks available for the Pi, so there's no need for us to recommend a particular model. (IMHO!)

Reply:
Yes, I'm not bothered about having any sort of link in there. The less off site links the better.